### ISSUE TITLE
add a page in the flow to collect name, city, & email

### ISSUE THREAD

**Issue by cydharrell** [codeforamerica/city-analytics-dashboard-setup#1]
This is a janky MVP version of us tracking who our users are, enabling Jack to set up conversations with them about the dashboard after they install it. Simple form with explanation of why we're collecting. Suggest placing it before the heroku account setup.

**Comment by cydharrell** [comment]
Oh, and the form can just be a Google form that collects to a sheet.

**Comment by phae** [comment]
If we're submitting it to a google spreadsheet, then it'll want to redirect there once you hit send. As such, I'd be inclined to make it the _last_ step for now, if that's OK?

**Comment by cydharrell** [comment]
Yes, that is fine! I think the text ought to be something like "Yay, you're all set up! We're interested in knowing who's using this so we can notify you of upgrades and possibly check in very occasionally with a few questions. Could you please tell us a little bit about yourself? (Of course we'll never share this information with anyone.)"

**Comment by phae** [comment]
Got it. :+1:

**Comment by migurski** [comment]
[Here’s what we grab currently](https://github.com/codeforamerica/city-analytics-dashboard-setup/blob/ba0450bc21169fde44c6bf7874e7741b88e2645b/schema.pgsql#L8), based on the Google connection.

**Comment by phae** [comment]
OK, a form has been added. @jmadans do you want to have a look at the copy for it and make some changes (I can help you out)?
